src/site/content/en/blog/requestvideoframecallback-rvfc/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: |
+  The requestVideoFrameCallback (rVFC) API
+subhead: |
+  Perform efficient per-video-frame operations on video.
+authors:
+  - thomassteiner
+date: 2020-06-29
+updated: 2020-06-29
+hero: hero.jpg
+alt: Film roll.
+description: |
+  The requestVideoFrameCallback method allows web authors to register a callback,
+  which runs in the rendering steps when a new video frame is sent to the compositor.
+tags:
+  - blog
+  - media
+  - capabilities
+---
+There's a new Web API on the block, defined in the
+[HTMLVideoElement.requestVideoFrameCallback()](https://wicg.github.io/video-rvfc/)
+specification.
+The `requestVideoFrameCallback()` method (rVFC) allows web authors to register a callback,
+which runs in the rendering steps when a new video frame 🎞 is sent to the compositor.
+This is intended to allow developers to perform efficient per-video-frame operations on video,
+such as video processing and painting to a canvas, video analysis,
+or synchronization with external audio sources.
+
+## Difference to requestAnimationFrame (rAF)
+
+Operations like drawing a video frame to a canvas via
+[`drawImage()`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage)
+made through this API will be synchronized as a best effort
+with the video playing on screen.
```

Comment:
I don't understand this sentence. Can you take another shot at it?

Reply:
Added that it's about the frame rate.